crawler/README.md

```diff
@@ -47,12 +59,25 @@ Next, information will be displayed for the Target Hostname, Server Gateway IP,
 
 As the script progresses, it will output feedback messages to the terminal indicating the ongoing operations.  Note that port-scanning typically consumes the majority of the scan time.
 
+## Integration Process
+Integration of the crawler Python file with other components posed several challenges. The crawler solution consists of helper files to download dependencies, pass information from file-to-file, and pipe feedback messages to the terminal in order to confirm integrity of important data (i.e. the username of the client currently logged in).
+
+1. Upon the repository tool's bootup, `login.html` prompts the user for their credentials.  This information is dynamically written to the UI's config.ini file for parsing and storage. 
```

Comment:
I'm not sure this is true. The login page should be for the user/pass for our app. The config file is for db credentials--these are not getting set dynamically in our current version ("Beyond scope").